tests/test_versions.py

```diff
@@ -1,20 +0,0 @@
-from typing import Optional
-
-import pytest
-
-from aidial_adapter_openai.utils.versions import compare_versions
-
-compare_versions_dataset = [
```

Comment:
Do we have an option to have some tests for mapping? For example, by mocking `ChatCompletion().acreate()`?

Reply:
Unfortunately, we haven't found a nice way to cover system variables with tests. As a temporary solution, I did manual testing.